### ISSUE TITLE
Injective exchange - Failed to check balance/start the strategy

### ISSUE THREAD

**Issue by nikspz** [hummingbot/hummingbot#6143]
Note: please be advised that when successfully added injective protocol wallet, balance command most like will not work and does not display balances. For workaround delete the gateway_connection.json file from conf folder

Steps to reproduce:

1. Clone and install the 6139 and clone/install gatewayPR55
2. yarn start the gateway(PR55) and the client so gateway is ONLINE
3. `gateway connect injective`
4. check balance
4.1 create/start pureMM strategy using injective

[logs_hummingbot.log](https://github.com/hummingbot/hummingbot/files/10942362/logs_hummingbot.log)
[clob.zip](https://github.com/hummingbot/hummingbot/files/10942356/clob.zip)
```
2023-03-10 18:42:16,129 - 19075 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: Can't instantiate abstract class GatewayCLOBSPOT with abstract methods _is_order_not_found_during_cancelation_error, _is_order_not_found_during_status_update_error
Traceback (most recent call last):
  File "/home/etozhe/batch6139/hummingbot/core/utils/async_utils.py", line 9, in safe_wrapper
    return await c
  File "/home/etozhe/batch6139/hummingbot/client/command/import_command.py", line 67, in import_config_file
    all_status_go = await self.status_check_all()
  File "/home/etozhe/batch6139/hummingbot/client/command/status_command.py", line 172, in status_check_all
    invalid_conns = await asyncio.wait_for(self.validate_required_connections(), network_timeout)
  File "/home/etozhe/miniconda3/envs/hummingbot/lib/python3.8/asyncio/tasks.py", line 483, in wait_for
    return fut.result()
  File "/home/etozhe/batch6139/hummingbot/client/command/status_command.py", line 95, in validate_required_connections
    connections = await UserBalances.instance().update_exchanges(self.client_config_map, exchanges=required_exchanges)
  File "/home/etozhe/batch6139/hummingbot/user/user_balances.py", line 136, in update_exchanges
    results = await safe_gather(*tasks)
  File "/home/etozhe/batch6139/hummingbot/core/utils/async_utils.py", line 22, in safe_gather
    return await asyncio.gather(*args, **kwargs)
  File "/home/etozhe/batch6139/hummingbot/user/user_balances.py", line 110, in update_exchange_balance
    return await self.add_exchange(exchange_name, client_config_map, **api_keys)
  File "/home/etozhe/batch6139/hummingbot/user/user_balances.py", line 86, in add_exchange
    market = UserBalances.connect_market(exchange, client_config_map, **api_details)
  File "/home/etozhe/batch6139/hummingbot/user/user_balances.py", line 45, in connect_market
    connector = connector_class(**init_params)
TypeError: Can't instantiate abstract class GatewayCLOBSPOT with abstract methods _is_order_not_found_during_cancelation_error, _is_order_not_found_during_status_update_error
```
![image](https://user-images.githubusercontent.com/83953535/224324005-3856b6a0-adae-45fe-95c0-8b1893017796.png)

Failed to start the strategy using injective, client showing error
![image](https://user-images.githubusercontent.com/83953535/224325285-726f7d4e-cddd-4b10-b119-dacd201ee51d.png)

**Comment by petioptrv** [comment]
Fixed in [this commit](https://github.com/hummingbot/hummingbot/pull/6139/commits/8ceead3e23c7432883b72c2dbbf64d138c6c79d0).

**Comment by petioptrv** [comment]
@nikspz, this issue should be fixed now.

**Comment by nikspz** [comment]
Yes, I think it's addressed in #6139 PR @petioptrv 
just wait for the merge
